test/e2e/framework/replication-controller.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2016 The prometheus-operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package framework
+
+import (
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/pkg/api/v1"
+	"os"
+)
+
+func createReplicationControllerViaYml(filepath string, f *Framework) error {
+	manifest, err := os.Open(filepath)
+	if err != nil {
+		return err
+	}
+
+	replicationController := v1.ReplicationController{}
```

Comment:
`var replicationController v1.ReplicationController` is the typical way to instantiate a zeroed object in Go.
With such a small function scope, i.e. not a lot of identifies to keep track of, one would probably go for a shorter name like `rc`.